src/libs/ReportUtils.ts

```diff
@@ -2031,6 +2038,18 @@ function getReportPreviewMessage(
     }
 
     const containsNonReimbursable = hasNonReimbursableTransactions(report.reportID);
+
+    const lastActorID = reportAction?.actorAccountID;
+
+    // if we have the amount in the originalMessage and lastActorID, we can use that to display the preview message for the latest request
+    if (originalMessage?.amount !== undefined && lastActorID && !isPreviewMessageForParentChatReport) {
+        const amount = originalMessage?.amount;
+        const currency = originalMessage?.currency ?? report.currency ?? '';
+        const amountToDisplay = CurrencyUtils.convertToDisplayString(Math.abs(amount), currency);
```

Comment:
In theory the expense report total or request could be negative value, so we should not use Abs here but more so invert the value if expense.

However, I think its not a concern for now

Reply:
@mountiny so can I leave it for now as it is?

Reply:
Yes, just noting